security/sshdo/files/macports.patch

```diff
@@ -0,0 +1,38 @@
+--- Makefile.orig	2023-06-19 15:23:20.000000000 +1000
++++ Makefile	2023-06-19 15:58:11.000000000 +1000
+@@ -58,8 +58,8 @@
+ 	@echo "make diff      - Show differences between source and installed versions"
+ 
+ test:
+-	@[ -z "`which python3 2>/dev/null | grep '^/'`" -a -n "`which python 2>/dev/null | grep '^/'`" ] && sed 's/env python3$$/env python/' < ./sshdo > ./sshdo2 && chmod 755 ./sshdo2 && mv ./sshdo2 ./sshdo || true
+-	@[ -z "`which python3 2>/dev/null | grep '^/'`" -a -n "`which python 2>/dev/null | grep '^/'`" ] && sed 's/env python3$$/env python/' < ./test_sshdo > ./test_sshdo2 && chmod 755 ./test_sshdo2 && mv ./test_sshdo2 ./test_sshdo || true
++	@[ -z "`which python@PYTHON_BRANCH@ 2>/dev/null | grep '^/'`" -a -n "`which python 2>/dev/null | grep '^/'`" ] && sed 's/env python3\.11$$/env python/' < ./sshdo > ./sshdo2 && chmod 755 ./sshdo2 && mv ./sshdo2 ./sshdo || true
++	@[ -z "`which python@PYTHON_BRANCH@ 2>/dev/null | grep '^/'`" -a -n "`which python 2>/dev/null | grep '^/'`" ] && sed 's/env python3\.11$$/env python/' < ./test_sshdo > ./test_sshdo2 && chmod 755 ./test_sshdo2 && mv ./test_sshdo2 ./test_sshdo || true
+ 	@./test_sshdo
+ 
+ check: test
+@@ -69,7 +69,7 @@
+ 	if [ ! -f $(DEST_ETCDIR)/$(SSHDOERS) ]; then cp $(SSHDOERS) $(DEST_ETCDIR); chmod 644 $(DEST_ETCDIR)/$(SSHDOERS); fi
+ 	if [ ! -f $(DEST_ETCDIR)/$(SSHDO_BANNER) ]; then cp $(SSHDO_BANNER) $(DEST_ETCDIR); chmod 644 $(DEST_ETCDIR)/$(SSHDO_BANNER); fi
+ 	cp $(BIN) $(DEST_BINDIR); chmod 755 $(DEST_BINDIR)/$(BIN)
```

Comment:
The `install` program exists to copy files and set their permissions, so probably most instances of `cp` (with or without `chmod`) in the install target could be replaced with `install` with the appropriate flags.

Reply:
That's true for some (admittedly most) operating systems. But on Solaris, the install program is very different. That's why I tend to use mkdir/cp/chown. It's more portable.

Is that OK, or do you need me to patch it?